app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutUploader.java

```diff
@@ -941,7 +942,7 @@ private void postDeviceStatus(NightscoutService nightscoutService, String apiSec
                     break;
                 case "Bridge":
                     battery_level = Pref.getInt("bridge_battery", -1);
-                    battery_name = DexCollectionType.getDexCollectionType().name();
+                    battery_name = DexCollectionService.getBestLimitterHardwareName();
```

Comment:
This method currently only works for LimiTTer

Reply:
The name of the function is misspelling. Getbestlimmitterhardwarename will return the real device name (like also bluereader, limmitter, bluecon, sweetreader)